03_smart_contract.md

```diff
@@ -296,447 +331,169 @@ fn get_balance(addr: Identity) -> u64 {
-    },
-    {
-      "owner": "0xa1184d77d0d08a064e03b2bd9f50863e88faddea4693a05ca1ee9b1732ea99b7",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0xb5566df884bee4e458151c2fe4082c8af38095cc442c61e0dc83a371d70d88fd",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0x9da7247e1d63d30d69f136f0f8654ee8340362c785b50f0a60513c7edbf5bb7c",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0x4b2ca966aad1a9d66994731db5138933cf61679107c3cde2a10d9594e47c084e",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0x26183fbe7375045250865947695dfc12500dcc43efb9102b4e8c4d3c20009dcb",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0x81f3a10b61828580d06cc4c7b0ed8f59b9fb618be856c55d33decd95489a1e23",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0x587aa0482482efea0234752d1ad9a9c438d1f34d2859b8bef2d56a432cb68e33",
-      "amount": "0x1000000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    },
-    {
-      "owner": "0x53a9c6a74bee79c5e04115a007984f4bddaafed75f512f68766c6ed59d0aedec",
-      "amount": "0x0004000000000000",
-      "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
-    }
-  ],
-  "contracts": [
-    {
-      "contract_id": "0x7777777777777777777777777777777777777777777777777777777777777777",
-      "code": "0x9000000994318e6e453f30e85bf6088f7161d44e57b86a6af0c955d22b353f91b2465f5e6140000a504d00205d4d30001a4860004945048076440001240400005050c0043d51345024040000",
-      "salt": "0x1bfd51cb31b8d0bc7d93d38f97ab771267d8786ab87073e0c2b8f9ddc44b274e"
-    }
-  ]
-},
-"consensus_parameters": {
-  "tx_params": {
-    "max_inputs": 255,
-    "max_outputs": 255,
-    "max_witnesses": 255,
-    "max_gas_per_tx": 1000000000,
-    "max_size": 17825792
-  },
-  "predicate_params": {
-    "max_predicate_length": 1048576,
-    "max_predicate_data_length": 1048576,
-    "max_message_data_length": 1048576,
-    "max_gas_per_predicate": 1000000000
-  },
-  "script_params": {
-    "max_script_length": 1048576,
-    "max_script_data_length": 1048576
-  },
-  "contract_params": {
-    "contract_max_size": 16777216,
-    "max_storage_slots": 131072
+  "chain_name": "Testnet",
+  "block_gas_limit": 1000000000,
+  "initial_state": {
+    "coins": [
+      {
+        "owner": "0x387c9580ba76a18c62d38edd400c0473429baa257b2570ee508c5061be25d035",
```

Comment:
explain the field to initialize some ETH for a wallet address here.

Also can we just add 1-2 owners here? or do we need all the entries.